src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -96,6 +96,10 @@
  * - Implement LB service forwarding (point 2c. in the doc's diagram).
  */
 
+/* We currently need this at the top of the file if we import some iomgr
```

Comment:
Don't we have the same restriction with Windows headers ? At the end, libuv trying to be a crossplatform library will have the same limitation as the whole set of platforms it supports, including Windows.